packages/modules/auth/src/providers/google.ts

```diff
@@ -1,13 +1,12 @@
 import { AuthenticationInput, AuthenticationResponse } from "@medusajs/types"
 import { AbstractAuthModuleProvider, MedusaError } from "@medusajs/utils"
-import { AuthIdentityService } from "@services"
 import jwt, { JwtPayload } from "jsonwebtoken"
 
 import { AuthorizationCode } from "simple-oauth2"
 import url from "url"
 
 type InjectedDependencies = {
-  authIdentityService: AuthIdentityService
+  authIdentityService: any
```

Comment:
Once this is moved to a package this will be injected as a parameter instead, so just typed it as any for now.